tests/sequence_labelling/preprocess_test.py

```diff
@@ -132,3 +131,24 @@ def test_should_select_features(self):
         features_length = len(preprocessor.features_indices)
         assert features_length == 1
         assert all_close(features_transformed, [[[1], [2], [3]]])
+
+    def test_serialize_to_json(self):
+        preprocessor = FeaturesPreprocessor(features_indices=[1])
+        word_preprocessor = WordPreprocessor(feature_preprocessor=preprocessor)
+
+        import tempfile
+        f = tempfile.NamedTemporaryFile(delete=True)
```

Comment:
pytest has the `tmpdir` fixture. I would definitely recommend using that for any test specific temporary file needs. It's interface is somewhat similar to pathlib's Path but not quite. (That is why I personally prefer to wrap it as a separate fixture that is a Path)

Reply:
I only realise now, that `pytest` has now a `tmp_path` fixture which is what I would use now.

Reply:
Could you share an example?

Reply:
no need, I updated the best I could in f84b8ae